src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -10,6 +10,7 @@ type __pthread_spin_t = __cpu_simple_lock_nv_t;
 pub type vm_size_t = ::uintptr_t;
 pub type lwpid_t = ::c_uint;
 pub type shmatt_t = ::c_uint;
+pub type timer_t = *mut ::c_void;
```

Comment:
Hm, seems it's defined as `int`, am I missing something?

https://github.com/NetBSD/src/search?q=_BSD_TIMER_T_